pkg/ingester/ingester.go

```diff
@@ -2285,10 +2292,10 @@ func (i *Ingester) getOrCreateTSDB(userID string, force bool) (*userTSDB, error)
 
 func (i *Ingester) blockChunkQuerierFunc(userId string) tsdb.BlockChunkQuerierFunc {
```

Comment:
[not a blocker] for the sake of following the same convention, should not this also return an `error`